package/Linux.mk

```diff
@@ -98,7 +98,9 @@ install:
 	$(V1) $(MKDIR) -p $(DESTDIR)$(datadir)
 	$(V1) $(MKDIR) -p $(DESTDIR)$(datadir)/applications
 	$(V1) $(MKDIR) -p $(DESTDIR)$(datadir)/pixmaps
-	$(V1) $(MKDIR) -p $(DESTDIR)$(udevdir)
+# Following line creates udev directory - however udevdir is not previously defined, nor is anything
+# installed to this location in the rest of this recipe
+#	$(V1) $(MKDIR)  $(DESTDIR)$(udevdir)
```

Comment:
We should be adding the udev rules really - I need to find where they are installed. I think I may have added them previously manually! I'll look at this a bit more.